.github/workflows/static-analysis.yml

```diff
@@ -0,0 +1,85 @@
+name: Static Analysis
```

Comment:
@Golbolco for safer merge. If you dont want to go through all the modified files. You can copy over only this workflow + tests. Then run prettier yourself on the master branch